docs/CalicoSwarm.md

```diff
@@ -0,0 +1,152 @@
+# Running calico-docker networking on a Docker Swarm.
+The following instructions configure a Docker Swarm cluster networked using Calico.  In this tutorial, we will do the following.
+- Configure Docker Swarm on a VM cluster.
+- Configure etcd and Calico on our cluster.
+- Provision workloads on the Swarm, and check they can communicate.
+
+## Prerequisites
+To complete this demo, make sure you have the following prerequisites.
+- Four Linux servers (VMs or bare-metal) with Docker 1.4 or later installed.  The servers should have IP connectivity to each other.
+  - 1 client (to control the Swarm)
+  - 1 Swarm manager (to manage the Swarm nodes)
+  - 2 Swarm nodes (to host our containers)
+
+## Installing Swarm on your cluster
+Our Docker Swarm will consist of 2 nodes and a Swarm manager.  We will use the fourth server as the client to control our Swarm.  We'll use the token-based discovery backend to configure our Swarm.
+
+First, let us create the token which will identify our swarm.  To do this, run the following on your client node:
+```
+docker pull swarm
+docker run --rm swarm create
+``` 
+
+The second command will print a unique cluster ID token which we must use to configure our Swarm.  We'll reference this value in the tutorial as ```<swarm_token>```. 
+Your token should look something like this: d435dc114a3dd89af2fae239257338ce
+
+Now that we've got a token, we can begin to configure our Swarm.  First, let's join each of our two Swarm nodes to the
+cluster.  Run the following commands on each node, replacing ```<swarm_token>``` with the token from above, and ```<node_ip>``` with the IP address of this node. 
+```
+docker run -d swarm join --addr=<node_ip>:2377 token://<swarm_token>
```

Comment:
Might be nice to setup $NODE_IP beforehand to use this.  Maybe set up all the env variables upfront.